lib/extension.js

```diff
@@ -1,11 +1,11 @@
 const path = require('path');
 
-const EXTRE = /^[.]?[^.]+([.].*)$/;
+const EXTRE = /(\.[^.]+)+$/;
```

Comment:
Agreed. I will investigate how to solve that. I was trying to use one regex only, but I guess it will not be possible. Do you have handy a regex that does this job ?